test/colony-network.js

```diff
@@ -295,4 +295,22 @@ contract("ColonyNetwork", accounts => {
       assert.equal(skillCount.toNumber(), 1);
     });
   });
+
+  describe("when getting a skill", () => {
```

Comment:
Can we move these tests to `meta-colony.js`? That's where all skills related functionality is held atm